requirements.txt

```diff
@@ -20,7 +20,6 @@ Cython                   # required
 omfit_classes            # required
 pexpect                  # required
 fortranformat            # required
-pygacode                 # required
```

Comment:
All these packages are only required to support machine mappings.
I can agree to make `machine` an extra and not a requirement.
Then we can remove `omfit_classes`, `pexpect`, `fortranformat` in addition to `pygacode`.